tox.ini

```diff
@@ -5,7 +5,7 @@ envlist = flake8, py3
 deps =
     -rdateparser_scripts/requirements.txt
     -rtests/requirements.txt
-    -rfuzzing/requirements.txt
+    atheris; python_version < '3.12'
```

Comment:
I added it as I got some errors in the pipeline for 3.12 and newer versions. And it is the reason why I removed the 
`-rfuzzing/requirements.txt` file @Gallaecio 
https://github.com/google/atheris/issues/82#issuecomment-2554979449